src/bid.rs

```diff
@@ -119,9 +114,9 @@ impl
         buf[128..192].copy_from_slice(&self.stealth_address.to_bytes());
         buf[192..224].copy_from_slice(&self.hashed_secret.to_bytes());
         buf[224..256].copy_from_slice(&self.c.to_bytes());
-        buf[256..264].copy_from_slice(&self.eligibility.to_le_bytes());
-        buf[264..272].copy_from_slice(&self.expiration.to_le_bytes());
-        buf[272..Self::SIZE].copy_from_slice(&self.pos.to_le_bytes());
+        buf[256..264].copy_from_slice(&self.eligibility.to_bytes());
```

Comment:
Maybe the next step would be to write a `Writer` for `dusk-bytes`?

Reply:
I'm actually on it. Dissambiguating types with Canon::write..